### PR TITLE
Arb ts next  cleanup / housekeeping

### PR DESCRIPTION
**Pull request by DZGoldman**
just little things

### CODE REVIEW

File: packages/arb-ts/src/lib/rollupProtocol/rollupProtocol.ts [hunk 1]
```diff
@@ -19,52 +19,16 @@
...
 import { NodeConfirmedEvent, NodeCreatedEvent } from '../abi/Rollup'
 import { EventFetcher } from '../utils/eventFetcher'
-import { ADDRESS_ALIAS_OFFSET } from '../dataEntities/constants'
 
 /**
- * General information about the current network state
+ * General information about protocol level actions, WIP
  */
-export class NetworkState {
+export class RollupProtocol {
```
Comment (yahgwai) [inline packages/arb-ts/src/lib/rollupProtocol/rollupProtocol.ts:29]: I think this class has been deleted now, maybe we need to merge arb-ts-next into this branch

Comment (DZGoldman) [reply packages/arb-ts/src/lib/rollupProtocol/rollupProtocol.ts:29]: ah

Comment (DZGoldman) [reply packages/arb-ts/src/lib/rollupProtocol/rollupProtocol.ts:29]: updated

**Comment by DZGoldman** [comment]
brought in the changes and fixed conflict from arb-ts-next @yahgwai

**Review by fredlacs — CHANGES_REQUESTED** [review]
I think we should delete the rollupState file.
Its not useful (and the file had already been deleted in https://github.com/OffchainLabs/arbitrum/pull/2069)

**Comment by fredlacs** [comment]
We should merge the latest `arb-ts-next` into `arb-ts-next--docs` since this seems to have some outdated stuff

**Review by fredlacs — APPROVED** [review]
LGTM

**Pull request merged**